src/reporter.ts

```diff
@@ -127,12 +138,22 @@ export function scrubSpans(resultSpans: any[]) {
   return resultSpans.filter((span) => safeExecute(scrubSpan, 'Failed to scrub span')(span));
 }
 
-export function getPrioritizedSpans(spans: any[], maxSendBytes: number): any[] {
+export function getPrioritizedSpans(
+  spans: any[],
+  maxSendBytes: number,
+  addEnrichmentSpan: boolean = true
```

Comment:
did i miss something, or this parameter is never used?

Reply:
Your correct, I left this by mistake here.
Removed it